fastlane/README.md

```diff
@@ -0,0 +1,30 @@
+fastlane documentation
+================
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```
+xcode-select --install
+```
+
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew install fastlane`
```

Comment:
Do we need this file at all? It's a little confusing since it suggest multiple things. Also the PR description says `bundle exec fastlane tests`, which I think is the way we want to go.

Maybe instead we can just have the commands to run test documented in the README or DEVELOPMENT.md?

Reply:
Eh not really! I kept it in because it was auto generated. I figure if we have a number of lanes, it could be helpful to see them and their documentation all in one place. But the main readme is also a fine place to keep that.

Reply:
Removed this file.